src/pages/settings/AboutPage.js

```diff
@@ -0,0 +1,168 @@
+import ROUTES from '../../ROUTES';
+import styles from '../../styles/styles';
+import Text from '../../components/Text';
+import CONST from '../../CONST';
+import {
+    Link, Eye, MoneyBag, Bug, NewWindow,
+} from '../../components/Icon/Expensicons';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import withLocalize, {
+    withLocalizePropTypes,
+} from '../../components/withLocalize';
+import compose from '../../libs/compose';
+import MenuItem from '../../components/MenuItem';
+import Logo from '../../../assets/images/expensify-cash.svg';
+import {version} from '../../../package.json';
+import openURLInNewTab from '../../libs/openURLInNewTab';
+import ONYXKEYS from '../../ONYXKEYS';
+
+
+const propTypes = {
+    // All reports shared with the user
+    reports: PropTypes.shape({
+        reportID: PropTypes.number,
+        reportName: PropTypes.string,
+    }).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+
+};
+
+const AboutPage = ({translate, reports}) => {
+    const [reportID, setreportID] = useState(null);
+
+    useEffect(() => {
+        const reportKey = _.findKey(reports, ['lastActorEmail', CONST.EMAIL.CONCIERGE]);
+        if (reportKey) {
+            setreportID(reports[reportKey].reportID);
+        }
+    }, [reports]);
+
+
+    const menuItems = [
+        {
+            translationKey: 'initialSettingsPage.aboutPage.appDownloadLinks',
+            icon: Link,
+            action: () => {
+                Navigation.navigate(ROUTES.SETTINGS_APP_DOWNLOAD_LINKS);
+            },
+        },
+        {
+            translationKey: 'initialSettingsPage.aboutPage.viewTheCode',
+            icon: Eye,
+            iconRight: NewWindow,
+            action: () => {
+                openURLInNewTab(CONST.GITHUB_URL);
+            },
+        },
+        {
+            translationKey: 'initialSettingsPage.aboutPage.viewOpenJobs',
+            icon: MoneyBag,
+            iconRight: NewWindow,
+            action: () => {
+                openURLInNewTab(CONST.UPWORK_URL);
+            },
+        },
+        {
+            translationKey: 'initialSettingsPage.aboutPage.reportABug',
+            icon: Bug,
+            action: () => {
+                if (reportID) { Navigation.navigate(ROUTES.getReportRoute(reportID)); }
+            },
+        },
+
+    ];
+
+
+    return (
+        <ScreenWrapper>
+            <HeaderWithCloseButton
+                title={translate('initialSettingsPage.about')}
+                shouldShowBackButton
+                onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
+                onCloseButtonPress={() => Navigation.dismissModal(true)}
+            />
+            <View style={styles.pageWrapper}>
+                <View style={[styles.settingsPageBody, styles.mb6]}>
+                    <Logo height={100} />
+                    <Text style={[styles.textLabel, styles.alignSelfCenter, {marginVertical: 10}]}>
```

Comment:
You should be able to use a utility style for the margin rather than using an inline style here. So for example, we might already have something like `mv-2` or `mv-3`. Check them out here: https://github.com/Expensify/Expensify.cash/blob/main/src/styles/utilities/spacing.js It looks like you might need to create the style you need for `mv-X` or alternatively, just use `mt-2 mb-2`. As a heads up, we do everything as an increment of the unit of 4. 

Can you also give this

Reply:
Thanks for the review @shawnborton, I have added `mv[1,2,3,4,5]`. I think this looks more maintainable and clean.